cmd/registry/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/docker/distribution/registry/storage/driver/filesystem"
 	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
 	_ "github.com/docker/distribution/registry/storage/driver/middleware/cloudfront"
+	_ "github.com/docker/distribution/registry/storage/driver/rados"
```

Comment:
Hmm... I don't get it: this file is guarded by the build tag, an empty package [`rados`](https://github.com/gierschv/distribution/blob/driver-rados/registry/storage/driver/rados/rados_none.go) is included when the build tag is not set.